samples/net/ocpp/overlay-ec.conf

```diff
@@ -0,0 +1,7 @@
+CONFIG_PSA_WANT_ALG_ECDH=y
```

Comment:
Do we need a separate overlay here, couldn't all required configs be placed in `overlay-tls13.conf`?